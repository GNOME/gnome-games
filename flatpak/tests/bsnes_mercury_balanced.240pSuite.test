[Retro Reftest]
Path=/bsnes_mercury_balanced/240pSuite
Core=/app/lib/libretro/bsnes_mercury_balanced_libretro.so
Medias=240pSuite.sfc;

[Options]

[Frame 120]
Video=bsnes_mercury_balanced.240pSuite.120.png
