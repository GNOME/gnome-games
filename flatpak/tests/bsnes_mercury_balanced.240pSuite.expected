/bsnes_mercury_balanced/240pSuite/Boot: OK
/bsnes_mercury_balanced/240pSuite/Options: OK
/bsnes_mercury_balanced/240pSuite/120/FastForward: OK
/bsnes_mercury_balanced/240pSuite/120/Run: OK
/bsnes_mercury_balanced/240pSuite/120/Video: OK
